qt/widgets/spectrumviewer/src/MatrixWSDataSource.cpp

```diff
@@ -242,7 +242,7 @@ std::vector<std::string> MatrixWSDataSource::getInfoList(double x, double y) {
 
   std::string x_label = "";
   Unit_sptr &old_unit = m_matWs->getAxis(0)->unit();
-  if (old_unit != 0) {
+  if (old_unit != nullptr) {
```

Comment:
As `shared_ptr` has conversion to bool this is probably better just as `if(old_unit)`?